web/src/locales/en/data.json

```diff
@@ -206,6 +206,7 @@
     "Admin": "Admin",
     "Affiliation URL": "Affiliation URL",
     "Affiliation URL - Tooltip": "The homepage URL for the affiliation",
+    "AI Assistant": "AI Assistant",
```

Comment:
Update i18n in this way: https://casdoor.org/docs/contributing#how-to-update-i18n-data